site/packages/aa-core/smart-account-client/actions/upgradeAccount.md

```diff
@@ -0,0 +1,43 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: upgradeAccount
+  - - meta
+    - name: description
+      content: Overview of the upgradeAccount method on ISmartAccountClient
+  - - meta
+    - property: og:description
+      content: Overview of the upgradeAccount method on ISmartAccountClient
+---
+
+# upgradeAccount
+
+Returns the address of the connected account. Throws error if there is no connected account.
+
+## Usage
+
+::: code-group
+
+```ts [example.ts]
+import { smartAccountClient } from "./smartAccountClient";
+// [!code focus:99]
+const address = await smartAccountClient.getAddress();
```

Comment:
This looks like it's missing the example of actually calling client.upgradeAccount